app/models/shelter.rb

```diff
@@ -6,4 +6,12 @@ class Shelter < ApplicationRecord
   def pet_count
     pets.count
   end
+
+  def no_pending? # needs heavy refactor, ruby doc site is down (~12/8/19 7am)
+    return true if pet_count == 0
+    pets.each do |pet|
+      return false if !pet.adoptable
+    end
+    true
```

Comment:
I think we can refactor this by using the `.any?` enumerable, something like this:

```
pets.any? { |pet| !pet.adoptable }
```